nodes/pods/nodes-pods-autoscaling-custom.adoc

```diff
@@ -20,6 +20,9 @@ The custom metrics autoscaler currently supports only the Prometheus, CPU, memor
 
 // For example, you can scale a database application based on the number of tables in the database, scale another application based on the number of messages in a Kafka topic, or scale based on incoming HTTP requests collected by {product-title} monitoring.
 
+:FeatureName: Autoscaling based on Apache Kafka metrics
```

Comment:
I was hoping there would be a way to combine the TP snippets to avoid the two warnings for similar (but not the same?) things as they are showing on the HTML output, but if one note is in the module and one in the assembly, I guess that wouldn't work. It's just a bit confusing as to how these two are different based on the limited info right before the two warnings. 

![Screenshot 2023-05-05 at 8 28 41 AM](https://user-images.githubusercontent.com/96192119/236457696-1f49c76a-45f8-4dac-9df1-138848793844.png)

Reply:
I  also changed the NOTE to a regular paragraph.